ci/github-script/get-teams.js

```diff
@@ -0,0 +1,83 @@
+const excludeTeams = [
+  /^voters.*$/,
+  /^nixpkgs-maintainers$/,
+  /^nixpkgs-committers$/,
+]
+
+module.exports = async ({ github, context, core, outFile }) => {
+  const withRateLimit = require('./withRateLimit.js')
+  const { writeFile } = require('node:fs')
+  const result = {}
+  await withRateLimit({ github, core }, async (stats) => {
+    /// Turn an Array of users into an Object, mapping user.login -> user.id
+    function makeUserSet(users) {
+      return (
+        users
+          .sort((user1, user2) => (user1.login > user2.login ? 1 : -1))
+          // biome-ignore lint/performance/noAccumulatingSpread: We don't care about time complexity
+          .reduce((acc, user) => ({ ...acc, [user.login]: user.id }), {})
+      )
+    }
+
+    /// Process a list of teams and append to the result variable
+    async function processTeams(teams) {
+      for (const team of teams) {
+        core.notice(`Processing team ${team.slug}`)
+        if (
+          !excludeTeams
+            .map((regex) => team.slug.match(regex) != null)
+            .includes(true)
+        ) {
+          const members = makeUserSet(
+            await github.paginate(github.rest.teams.listMembersInOrg, {
+              org: context.repo.owner,
+              team_slug: team.slug,
+            }),
```

Comment:
I believe my comment elsewhere, to reduce duplication in the .json file, would only require this:


```suggestion
              org: context.repo.owner,
              team_slug: team.slug,
              role: 'member',
            }),
```

(ofc + relevant nix code)